consensus/polybft/consensus_runtime.go

```diff
@@ -586,9 +586,7 @@ func (c *consensusRuntime) buildBundles(commitment *Commitment, commitmentMsg *C
 
 	var bundleProofs []*BundleProof
 
-	startBundleIdx := commitmentMsg.GetBundleIdxFromStateSyncEventIdx(stateSyncExecutionIndex)
```

Comment:
What do you think about removing `GetBundleIdxFromStateSyncEventIdx`, since it is being used only in the unit tests for assertions.